### DISCUSSION TITLE
What is the purpose of designing label ceph.rook.io/do-not-reconcile?

### DISCUSSION THREAD

**Discussion by haoxiaoci** [rook/rook#14767] [Q&A] [answered]
Could you provide a simple example of the scenario it addresses?  I want to see if it applies to my situation.

**Comment by microyahoo** [comment, marked as answer]
> To allow for advanced maintenance and troubleshooting of Ceph daemons, the label “ceph.rook.io/do-not-reconcile” is now respected for all Ceph daemons. If this label is found on a daemon while the operator is reconciling, the daemon will be skipped. Previously, this label was only available to mons and OSDs.

https://blog.rook.io/rook-v1-13-storage-enhancements-c551f62892bb
